### ISSUE TITLE
question about scheduler state

### ISSUE THREAD

**Issue by WangKK1996** [yhenon/pytorch-retinanet#155]
Hi, thanks for your work.
In your code, you use scheduler and set patience on training loss, is that correct?  During traing, the model tend to overfit the training set, why not use focal loss on evalution dataset?
